uint8 FeatureType
string UUID
string Name
string Description
